net/mwan3/files/usr/sbin/mwan3rtmon

```diff
@@ -7,6 +7,7 @@ LOG="logger -t $(basename "$0")[$$] -p"
 
 clean_up() {
 	$LOG notice "Stopping mwan3rtmon..."
+	mwan3_unlock "service" "mwan3rtmon"
```

Comment:
I think this could free a lock not owned by mwan3rtmon. When the process exits, if it is holding the lock it should be released anway.